File: test/qr.erl
-module(qr).
-export([main/1]).

main(_) ->
    QR = qrcode:encode(<<"unsandbox-qr-ok">>),
    Dim = element(4, QR),
    io:format("QR:unsandbox-qr-ok:ROWS:~p~n", [Dim]),
    halt().

File: test/qr.py
#!/usr/bin/env python3
import qrcode
q = qrcode.QRCode(border=0)
q.add_data("unsandbox-qr-ok")
q.make()
rows = len(q.get_matrix())
print(f"QR:unsandbox-qr-ok:ROWS:{rows}")
